documentation/source/users/rmg/installation/anacondaDeveloper.rst

```diff
@@ -26,15 +26,15 @@ Installation by Source Using Anaconda Environment for Unix-based Systems: Linux
       Note that you should reinitialize or restart your terminal in order for the changes to take effect, as the installer will tell you.
 
 #. There are a few system-level dependencies which are required and should not be installed via Conda. These include
-   `Git <https://git-scm.com/>`_ for version control, `GNU Make <https://www.gnu.org/software/make/>`_, and the C and C++ compilers from the `GNU Compiler Collection (GCC) <https://gcc.gnu.org/>`_ for compiling RMG.
+   `Git <https://git-scm.com/>`_ for version control, `GNU Make <https://www.gnu.org/software/make/>`_, and the C and C++ compilers from the `GNU Compiler Collection (GCC) <https://gcc.gnu.org/>`_ for compiling RMG. For WSL users, `libxrender1 <https://packages.debian.org/sid/libxrender1>`_ may also need to be installed.
 
    For Linux users, you can check whether these are already installed by simply calling them via the command line, which
    will let you know if they are missing. To install any missing packages, you should use the appropriate package manager
    for your system.
 
    a. On Ubuntu and Debian the package manager is ``apt`` ::
 
-       sudo apt install git gcc g++ make
+       sudo apt install git gcc g++ make libxrender1
```

Comment:
I think this is still worth getting into the docs!

Reply:
I cleaned up the commit history of this branch and updated it. Should be good to merge now?

Reply:
This LGTM! I'll approve, and let's get a review from Richard as well.